ml-agents/mlagents/trainers/reward_providers/__init__.py

```diff
@@ -0,0 +1,15 @@
+from mlagents.trainers.reward_providers.base_reward_provider import (  # noqa F401
```

Comment:
Should all of these be put in a torch-specific sub-folder?

Reply:
Yes, I wanted to not have dependencies on other files for now, but these will definitely not stay here.